package.json

```diff
@@ -36,7 +37,9 @@
     "moment": "^2.22.2",
     "ora": "^3.0.0",
     "prompt-confirm": "^2.0.4",
+    "typedarray-to-buffer": "^3.1.5",
     "web3": "1.0.0-beta.35",
+    "yaeti": "^1.0.2",
```

Comment:
why is this needed?

Reply:
It crashed and required to install that for some reason

Reply:
What crashed and why?

Reply:
internal/modules/cjs/loader.js:589
    throw err;
    ^

Error: Cannot find module 'typedarray-to-buffer'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:587:15)
    at Function.Module._load (internal/modules/cjs/loader.js:513:25)
    at Module.require (internal/modules/cjs/loader.js:643:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/Users/benkaufman/Documents/Work/DAOstack/Migration/node_modules/web3-providers-ws/node_modules/websocket/lib/W3CWebSocket.js:18:16)
    at Module._compile (internal/modules/cjs/loader.js:707:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:718:10)
    at Module.load (internal/modules/cjs/loader.js:605:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:544:12)
    at Function.Module._load (internal/modules/cjs/loader.js:536:3)

Reply:
When running `npm run migrate`

Reply:
and it didn't happen before this PR?

Reply:
It happened when I started working on the PR

Reply:
So something in this PR requires those package?

Reply:
After running npm install again it worked, strange.
Removed that.